js/wizard.js

```diff
@@ -1,5 +1,10 @@
 $(document).ready(function() {
 	
+	//Featherlight lightbox loads when page loads	
+	window.onload = function() {
```

Comment:
Looks like you may have created a branch from something other than gh-pages. This looks like code before your 2nd commit to pull request #18. Let's sit down together and take a look tomorrow morning.